src/components/embed_pages/FaceToFaceClassInfoEmbedPage.ts

```diff
@@ -15,7 +15,7 @@ export const data = new EmbedPage(
     [
         {
             name: 'Aula',
-            value: '305',
+            value: `${AlgoBot.dateFromISO(event.start.dateTime, event.start.timeZone).includes("Lunes") ? "313" : "400"}`,
```

Comment:
En este caso podes sacar la interpolacion de string porque las dos opciones ("313" y "400") son strings.

```suggestion
            value: AlgoBot.dateFromISO(event.start.dateTime, event.start.timeZone).includes("Lunes") ? "313" : "400",
```